src/stories/ObjectInline.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import {StoryFn} from '@storybook/react';
+
+import {ObjectInline as ObjectInlineBase, ObjectSpec, SpecTypes} from '../lib';
+
+import {InputPreview} from './components';
+
+export default {
+    title: 'Object/Inline',
```

Comment:
I would rename it to objectInline - as objectValue 
and accordingly in the config object_inline

Reply:
But for `ObjectBase` the name is `base`. I named it according to this logic.